src/converter/one/convert/Arguments.java

```diff
@@ -134,4 +134,20 @@ private static long parseTimestamp(String time) {
 
         return cal.getTimeInMillis();
     }
+
+    public String getValueType() {
+        if (nativemem) return "malloc";
+        if (alloc || live) return "allocations";
+        if (lock) return "locks";
+        return "cpu";
+    }
+
+    public String getSampleUnits() {
+        return "count";
+    }
+
+    public String getTotalUnits() {
+        if (nativemem || alloc || live) return "bytes";
+        return "nanoseconds";
+    }
```

Comment:
`Arguments` is a parser and holder for converter options, it is not supposed to contain business logic for interpreting options. `JfrConverter` is a better place for utility functions shared between different converters.

Reply:
97cbf40667345788c5b9f8e58395b80007a924eb